pmo/order/new/new.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @module {Module} new <pmo-order-new>
```

Comment:
Oh yeah, that's right. 

Sent from my iPhone

> On Aug 10, 2015, at 9:10 AM, Kevin Phillips <notifications@github.com> wrote:
> 
> In pmo/order/new/new.js:
> 
> > @@ -1,3 +1,7 @@
> > +/**
> > + * @module {Module} new <pmo-order-new>
> This comment is for the entire file so that it can tie together the ViewModel and the component constructor to this module and correctly nest the documentation.
> 
> —
> Reply to this email directly or view it on GitHub.
>